Package.resolved

```diff
@@ -64,6 +64,15 @@
           "version": "1.4.1"
         }
       },
+      {
+        "package": "FigSwiftArgumentParser",
```

Comment:
I'm not sure if this should be a direct dependency for Tuist. Could we create this whole integration as a plugin?

Reply:
@shahzadmajeed for this to work, Figs need to be integrated with every command in the CLI.
I am not sure how this could work as a plugin.

Reply:
@danyf90 the way to generate the spec is by running the following command:
```bash
   swift run tuist help --generate-fig-spec >> ../autocomplete/src/tuist.ts
```

where autocomplete is Figs repo, which then, in turn, requires to [push](https://fig.io/docs/guides/integrating/push-to-autocomplete-action) the spec into their repo.

Reply:
I don't think we should add a dependency to tuist just to generate something that is not needed by tuist itself.

I think it should either be generated from the autocomplete script if possible, or we should have a fork which adds this change just for the purpose of publishing the fig spec